### PR TITLE
Use parallel routes for permission on admin panel

### PR DESCRIPTION
**Pull request by ghost**
User permission functions in appwrite lib

### CODE REVIEW

File: apps/admin-panel/app/layout.tsx [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
-  }, []);
+  }, [getUser]);
 
   return (
     <html lang="en">
       <Head>
         <title>Welcome to admin-panel!</title>
       </Head>
-      <body>{!loading && children}</body>
+      <body>
```
Comment (dan-jones-nimble) [inline apps/admin-panel/app/layout.tsx]: probs needs another look

File: apps/admin-panel/app/login/page.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default function LoginPage() {
         password: credentials.password,
       });
       setError('');
-      push('/');
+      window.location.reload();
```
Comment (dan-jones-nimble) [inline apps/admin-panel/app/login/page.tsx:36]: wanna keep this?

File: libs/appwrite/src/lib/permissions/getUserPermissions.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { getAccount } from '../auth';
+import { getTeamMembership } from '../memberships';
+
+export const getUserPermissions = async ({
```
Comment (dan-jones-nimble) [inline libs/appwrite/src/lib/permissions/getUserPermissions.ts]: whats stopping you from just defining arguments like 
```
const getUserPermissions = async (teamName: string) => {
... 
```

Comment (ghost) [reply libs/appwrite/src/lib/permissions/getUserPermissions.ts]: More preference but I’ve changed it

**Review by dan-jones-nimble — APPROVED** [review]

**Pull request merged**